packages/builder/src/components/backend/DataTable/modals/CreateEditColumn.svelte

```diff
@@ -159,7 +160,7 @@
   async function saveColumn() {
     savingColumn = true
     if (errors?.length) {
-      return
+      return keepOpen
```

Comment:
This particular case didn't return `false` previously - are we still ok to update this to change the return value?

Reply:
Nope, you're right, mistake on my part.